### PR TITLE
Create  required documentation

### PR DESCRIPTION
**Pull request by nancyalaswad90**
Include brief documentation in Arabic about the translation process of the project in Crowdin.

### CODE REVIEW

**Comment by nancyalaswad90** [comment]
Include brief documentation in Arabic about the translation process of the project in Crowdin.

File: docs/doc.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.
+
+> ## تجربتي مع موقع كرودن crowdin للترجمة  و التعريب
```
Comment (BatoolMM) [inline docs/doc.md]: Crowdin is supposed to be written:
 (Crowdin) كراودن

File: docs/doc.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+.
+
+لطالما سعيت لإيجاد توضيحات لبعض المفاهيم والفقرات في مجالات الدراسات التي تشمل مجال  تكنولوجيا المعلومات ومجال المختبرات الطبية، وكنت الجأ دائما الى ترجمة جوجل والتي اعتبرتها المصدر المنطقي والمفيد لفهم وتبسيط المراجع والابحاث الاجنبية.
+
+
+الى ان اشتركت مع برنامج Outreachy والذي اتاح لي التعرف على عالم كامل من الفرص ، ومنها وجدت فرصة قيمة لفهم فكرة [التوطين](https://ar.motionpoint.com/resources/learn-todays-best-practices-for-website-translation/) **والذي يعتبر نوع من الترجمة الدقيقة التي تراعي الفروق الدقيقة والاختلافات الثقافية.**
+
+
+
+ومن خلال برنامج العلوم المفتوحة لمجتمع السعودية [Open Science Community Saudi Arabia](https://form.jotform.com/220362171812446) اشتركت بموقع Crowdin، والذي يعتبر من أفضل المواقع للترجمة والتعريب التي تعاملت معها، كما عرفت ايضا انه موقع معتمد لدى كثير من الشركات منها ميكروسوفت و غوغل، حيث تقوم الشركات بوضع ملفات اللغة الخاصة بمواقعها او برامجها ومن ثم طلب تعريبها من الناس.
```
Comment (BatoolMM) [inline docs/doc.md]:
```suggestion
ومن خلال مجتمع العلوم المفتوحة في المملكة العربية السعودية [Open Science Community Saudi Arabia](https://osc-ksa.com/) شاركت في  منصة Crowdin، والذي يعتبر من أفضل المنصات للترجمة والتعريب التي تعاملت معها، كما عرفت ايضا انه موقع معتمد لدى كثير من الشركات منها ميكروسوفت و غوغل، حيث تقوم الشركات بوضع ملفات اللغة الخاصة بمواقعها او برامجها ومن ثم طلب تعريبها من الناس.
```

File: docs/doc.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+اما طريقة عمل الموقع فهي سهلة، حيث نقوم بالتسجيل فيه من خلال حساباتنا مثل  فيس بوك او تويتر، وعند تفعيل الحساب نقوم باختيار أحد مشاريع ومن ثم البدء في تعريبها، من الجميل فعلا ان نجد موقع يدعم اللغة العربية. 
+
+
+وبالنسبة لخطوات العمل فهي كالتالي :  
+
+> - عند الدخول الى الموقع نذهب الى اللغة العربية.
+
+ ![image](https://user-images.githubusercontent.com/36210723/163408194-07c18633-920b-49b3-a844-68851d0c0c76.png)
+
+> - حيث نجد في هذه النافذة جميع الملفات الموجودةالتي يمكن ترجمتها.
```
Comment (BatoolMM) [inline docs/doc.md]:
```suggestion
> - حيث نجد في هذه النافذة جميع الملفات الموجودة التي يمكن ترجمتها. 
```

File: docs/doc.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+> - عند الدخول الى الموقع نذهب الى اللغة العربية.
+
+ ![image](https://user-images.githubusercontent.com/36210723/163408194-07c18633-920b-49b3-a844-68851d0c0c76.png)
+
+> - حيث نجد في هذه النافذة جميع الملفات الموجودةالتي يمكن ترجمتها. 
+
+<img width="1416" alt="160449198-05aee1ba-cdfb-4e9c-9f41-d17ca275a229" src="https://user-images.githubusercontent.com/36210723/163408928-0df2aad3-eafa-429c-b1ef-f9ac4f0a7514.png">
+
+
+> - نختار احد المشاريع التي نهتم به ثم نقوم بترجمته ولكن ليس ترجمة حرفية، فنحن نقوم هنا بالتوطين بمعنى نعمل على  ترجمة المشروع بلغة عربية تتلاءم مع مفاهيم وثقافات البدلدان العربية بحيث يقدر كل باحث قام بالاطلاع على المشروع المقدم من قبلنا بفهم الافكار الموجودة فيه بسلاسة ووضوح.
```
Comment (BatoolMM) [inline docs/doc.md]:
```suggestion
> - نختار احد المشاريع التي نهتم به ثم نقوم بترجمته ولكن ليس ترجمة حرفية، فنحن نقوم هنا بالتوطين بمعنى نعمل على  ترجمة المشروع بلغة عربية تتلاءم مع مفاهيم وثقافات البلدان العربية بحيث يقدر كل باحث قام بالاطلاع على المشروع المقدم من قبلنا بفهم الافكار الموجودة فيه بسلاسة ووضوح.
```

File: docs/doc.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+<img width="1421" alt="160450765-49765d54-8d26-419a-bf5b-de20c2e9c9de" src="https://user-images.githubusercontent.com/36210723/163409667-73d0453f-61c7-4789-b169-0817c72bc829.png">
+
+
+.
+
+> ### وللتوضيح اكثر ارجو الالطلاع على الرسوم المتحركة gif  ادناه، حيث نرى طريقة الترجمة لاسطر الملف ترجمة مفهومة ونرى ايضا وجود مشرفين يدعمون المترجمون في فهم اسلوب وخطوات العمل المطلوب.
```
Comment (BatoolMM) [inline docs/doc.md]:
```suggestion
> وللتوضيح اكثر ارجو الالطلاع على الرسوم المتحركة gif  ادناه، حيث نرى طريقة الترجمة لاسطر الملف ترجمة مفهومة ونرى ايضا وجود مشرفين يدعمون المترجمون في فهم اسلوب وخطوات العمل المطلوب.
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

File: docs/doc.md (hunk 1 above)
Comment (nancyalaswad90) [reply docs/doc.md]: done

**Review by nancyalaswad90 — COMMENTED** [review]
Thanks for your feedback

File: docs/doc.md (hunk 1 above)
Comment (BatoolMM) [reply docs/doc.md]: Throughout all documents, it should be:
(Crowdin) كراودن

Comment (nancyalaswad90) [reply docs/doc.md]: ** I made the changes above **

.

**Review by nancyalaswad90 — COMMENTED** [review]
I approve all the suggestion feedback , thanks @BatoolMM  for your support

File: docs/doc.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+.
+
+![xx](https://github.com/Open-Science-Community-Saudi-Arabia/ols3-cohort-talks-and-transcripts/raw/main/imgs/crowdin-approve.gif)
+.
+
+
+ **من تجربتي اعلاه، تعلمت قيمة الترجمة غير الحرفية `التوطين`** وتعلمت استخدام المواقع التي تعتمد التوطين بشكل يضمن لي الاستفادة مما تم ترجمته وايضا القدرة على افادة غيري من الطلاب والباحثين في فهم أعمق للدرسات الاجنبية القيمة خصوصا الدراسات في مجال المختبرات الطبية وامجال تكنولوجيا المعلومات .  
+ 
+ **ومن مميزات** برنامج العلوم المفتوحة لمجتمع السعودية ايضا التواصل مع مشرفين مثل الدكتورة batool@liverpool.ac.uk التي تقوم بتوجيهنا لاجراء خطوات الترجمة بشكل صحيح ومنظم، حتى يتنج عمل بمستوى متقدم.
```
Comment (BatoolMM) [inline docs/doc.md]:
```suggestion
 **ومن مميزات** برنامج العلوم المفتوحة لمجتمع السعودية ايضا التواصل مع مشرفين مثل [بتول المرزوق](https://batool-almarzouq.netlify.app/) التي تقوم بتوجيهنا لاجراء خطوات الترجمة بشكل صحيح ومنظم، حتى يتنج عمل بمستوى متقدم.
```

**Review by BatoolMM — APPROVED** [review]

**Comment by BatoolMM** [comment]
Thank you @nancyalaswad90 for this review!

**Comment by nancyalaswad90** [comment]
.

>  **Thank you so much @BatoolMM ,  I appreciate your support all these days .**

.

**Pull request merged**